packages/@sanity/components/src/toggles/Checkbox.js

```diff
@@ -38,19 +38,29 @@ export default class Checkbox extends React.Component {
 
   setInput = el => {
     this._input = el
+
+    if (typeof value === 'undefined' && el) {
```

Comment:
Could this be moved to `componentDidMount` instead? Personally that is the first place I'd look for it and expect it to be. And you don't have to worry about `el` (`this._input`) being null either.

Reply:
This applies to Switch as well

Reply:
👍